pkg/auth/api/api.go

```diff
@@ -69,16 +72,19 @@ func WithLogger(logger *zap.Logger) Option {
 }
 
 type authService struct {
-	oidc                *oidc.OIDC
+	issuer              string
+	audience            string
 	signer              token.Signer
 	accountClient       accountclient.Client
+	verifier            token.Verifier
 	googleAuthenticator auth.Authenticator
 	opts                *options
 	logger              *zap.Logger
 }
 
 func NewAuthService(
-	oidc *oidc.OIDC,
+	issuer string,
+	audience string,
```

Comment:
These two params didn't be used inside the `NewAuthService` fuction.

```go
return &authService{
		issuer:        issuer,
		audience:      audience,
		signer:        signer,
		accountClient: accountClient,
		verifier:      verifier,
		googleAuthenticator: google.NewAuthenticator(
			&config.GoogleConfig, signer, logger,
		),
		opts:   &options,
		logger: logger,
	}
```

Reply:
Fixed. Thanks!